index.js

```diff
@@ -1 +1,119 @@
 // implement your API here
+const express = require('express');
+const cors = require('cors');
+
+const users = require('./data/db.js');
+
+const server = express();
+
+server.use(express.json());
+server.use(cors());
+
+server.get('/', (req, res) => {
+  res.send('Get is working!!!');
+});
+
+server.get('/users', (req, res) => {
+  users
+    .find()
+    .then(users => {
+      res.status(200).json(users);
+    })
+    .catch(err => {
+      res
+        .status(500)
+        .json({ error: 'The users information could not be retrieved.' });
```

Comment:
You can keep it on one line.

Reply:
My linter moved it since it was longer than X chars